build.assets/Dockerfile-fpm

```diff
@@ -0,0 +1,29 @@
+FROM ubuntu:18.04
+# Runtime deps. Build deps go in the build or test containers
+WORKDIR /src
+
+RUN apt-get update \
+	&& apt-get -y dist-upgrade \
+	&& apt-get install --no-install-recommends -y \
+	ruby rubygems rubygems-integration \
+	bsdtar \
+	cpio \
+	debsigs \
+	pacman \
+	rpm  \
+	squashfs-tools \
+	xz-utils \
+	zip \
+  gcc \
+  make \
+  ruby-dev \
+  libc-dev \
```

Comment:
Formatting here is weird as it's part of the same command but lacking the initial spaces.